flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js

```diff
@@ -766,7 +766,54 @@ var A320_Neo_UpperECAM;
                                         action: "MAN ON"
                                     },
                                 ]
-                            }
+                            },
+                            {
+                                message: "EXCES RESIDUAL PR",
+                                level: 2,
```

Comment:
This is actually a level 3 message with CRC:

```suggestion
                                level: 3,
```